packages/sanity/src/desk/panes/document/documentPanel/header/DocumentHeaderTitle.tsx

```diff
@@ -21,12 +23,24 @@ export function DocumentHeaderTitle(): ReactElement {
   }
 
   if (!documentValue) {
-    return <>New {schemaType?.title || schemaType?.name}</>
+    return (
+      <>
+        {t('header.document-header-title.new.text', {item: schemaType?.title || schemaType?.name})}
```

Comment:
Would prefer if this had a more specific name
```suggestion
        {t('header.document-header-title.new.text', {schemaType: schemaType?.title || schemaType?.name})}
```